packages/types-algosdk/index.d.ts

```diff
@@ -555,11 +555,26 @@ export interface SSCParams {
   'local-state-schema': SSCSchemaConfig;
 }
 
+export interface AssetParams {
+  creator: string;
+  total: number;
+  decimals: number;
+  defaultFrozen: string;
+  unitName: string;
+  name: string;
+  url: string;
+  metadataHash: string;
+  manager: string;
+  reserve: string;
+  freeze: string;
+  clawback: string;
+}
```

Comment:
I think in `algosdk`, defaultFrozen is represented as 'default-frozen'. Can you please confirm?
If this was intentional (using our custom types for accounts), then i don't think `AssetParams` should be in `types/algosdk`. Can we move to `types/algorand-js`?

Reply:
yes, i was matching it from constructor. I think updating here is better. Thanks for catching